python/tutorials/06-fused-attention.py

```diff
@@ -571,7 +571,7 @@ def test_op(Z, H, N_CTX, D_HEAD, causal, dtype=torch.float16):
     assert torch.allclose(ref_out, tri_out, atol=1e-2, rtol=0)
     rtol = 0.0
     # Relative tolerance workaround for known hardware limitation of MI200 GPU.
-    # For detailss see https://pytorch.org/docs/stable/notes/numerical_accuracy.html#reduced-precision-fp16-and-bf16-gemms-and-convolutions-on-amd-instinct-mi200-devices
+    # For details see https://pytorch.org/docs/stable/notes/numerical_accuracy.html#reduced-precision-fp16-and-bf16-gemms-and-convolutions-on-amd-instinct-mi200-devices
     if torch.version.hip is not None and triton.runtime.driver.active.get_current_target()[1] == "gfx90a":
```

Comment:
Probably not directly from this pull request but here we can do `target = triton.runtime.driver.active.get_current_target()` and `if target[0] == "hip" and target[1] == "gfx90a"` to use one API for it.